### PR TITLE
Documentation for -a- group, acquiring lake centers for GEE pull

### LINKED ISSUE #14: Supplemental: Buffer distance
30m, 60, 90 etc. variance should be ~the same.

compare to the sample locations?

### LINKED ISSUE #15: Supplemental: use of POI
Just a quick comparison of how similar.

### PR DESCRIPTION
**Pull request by steeleb**
Hey Katie, MR, and MB -

Katie, tagging you for review since MB is out, but also tagging MR since this is the first round of documentation and I want to make sure it's on the right path for submission and setting you up for success in writing, and MB for consistency. MB, please review the rendered text, even after they have been merged to make sure it is in line with the earlier sections of ASv2. 

Most of the LOC here are .html, but there are also a few tweaks to the pipeline as a sanity check (specifically, making sure that the points are inside the polygon area of the filtered waterbodies. 

KW, please to a thorough read of the bookdown (I'll send the rendered .html to everyone in Teams), make sure links work, etc etc. If there are things that are unclear, please let me know and if there is additional documentation that you think is necessary to describe the -a- group of targets, please say so!

As a reminder, we had previously talked about doing a direct comparison of the POI locations between v1 and v2, but for a number of reasons, MR and I decided that was a bit out of scope. Same with playing with buffer distances. We're going to keep things as is and respond to feedback from reviewers.

Closes #14, Closes #15

### CODE REVIEW

File: bookdown/01-Introduction.Rmd [hunk 1]
```diff
@@ -43,44 +48,43 @@ code, or a target's dependencies have changed (or become "outdated" in {targets}
 speak).
 
 **lakeSR** is broken down into functional groups of targets, listed below with a
-brief summary about what task(s) each group completes.
```
Comment (kathryn-willi) [inline bookdown/01-Introduction.Rmd:46]: "functional group" == "a list of functions performed in specific order"?

**Review by kathryn-willi — APPROVED** [review]
Hey B - There were a couple places I thought might be helpful to clarify (see comments); I also saw several instances where your citation needs `[ ]` instead of `{ }` that you might want to double check! Not sure I got all of 'em. Otherwise looks great!

**Comment by steeleb** [comment]
you are a rockstar, KW!

File: bookdown/02-Data_Acquisition_Locations.Rmd [hunk 2]
```diff
@@ -3,149 +3,222 @@ editor_options:
...
+  tm_borders() +
+  tm_shape(WI_HL) +
+  tm_fill("lightblue") +
+  tm_borders() +
+  tm_shape(WI_HL_DP) +
+  tm_dots("black")
+
+```
+
+```{r v2-WI-lakes, fig.cap="Wisconsin lakes included in AquaSat v2, defined by the NHDPlusV2 data product."}
```
[the remark this answers is not in the document]
Comment (steeleb) [reply bookdown/02-Data_Acquisition_Locations.Rmd:216]: Updating to:

Deepest point of Wisconsin lakes included in AquaSat v1, defined by the Hydrolakes data product. Deepest point for each lake is indicated with a black dot.

Point of inaccessibility for Wisconsin lakes included in AquaSat v2, defined by the NHDPlusV2 data product. Point of inaccessibility for each lake is indicated with a black dot.

Does that read a bit better?

File: bookdown/01-Introduction.Rmd [hunk 3]
```diff
@@ -29,9 +32,11 @@ Together the Landsat archive of **lakeSR** is referred to as
 satellite data incorporated into the **lakeSR** product in the future.
```
[the remark this answers is not in the document]
Comment (steeleb) [reply bookdown/01-Introduction.Rmd:32]: i'm going to drop this until we have incorporated any other satellites. this was part of the original AS naming conventions, but it seems like overkill until there are other products.

File: bookdown/01-Introduction.Rmd (hunk 1 above)
Comment (steeleb) [reply bookdown/01-Introduction.Rmd:46]: kind of... more like functions that are grouped together to perform a specific task.

Edited to: 

lakeSR is broken down into _groups of targets that perform specific tasks_, listed below with a brief summary about what task(s) each group completes. This particular workflow incorporates both R and Python programming languages to complete these tasks.

**Pull request merged**